<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterTourist</class>
 <widget class="QDialog" name="RegisterTourist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register Tourist</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/images/d.jpg) ;</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>711</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Primer Nombre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtFirstName">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Primer Apellido</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtMiddleName">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Segundo Apellido</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtLastName">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Pasaporte ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtPassportNo">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Contacto</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtContactNo">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Direcci√≥n Permanente</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtPermanentAddress">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="textemail">
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Elija el paquete</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="cmbChoosePackage">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="loadPakcage">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
background-color: #000000;
background-color: #0000FF;



</string>
      </property>
      <property name="text">
       <string>Cargar Paquete</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QTextEdit" name="txtPackageDescription">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/images/b.jpg) ;</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>Este paquete contiene:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QPushButton" name="btnSave">
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QPushButton" name="btnReset">
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;

</string>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8A2BE2;
background-color: transparent;

border: none;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-style:normal; color:#ffffff;&quot;&gt;Gesti√≥n de Reservas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>textEdit</zorder>
  <zorder>formLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
